src/GuestConfiguration/GuestConfiguration/Common/GuestConfigurationCmdletBase.cs

```diff
@@ -49,6 +51,11 @@ public abstract class GuestConfigurationCmdletBase : AzureRMCmdlet
         /// </summary>
         private IPolicyClient _policyClient;
 
+        /// <summary>
+        /// Policy insights client
+        /// </summary>
+        private IPolicyInsightsClient _policyInsightsClient;
```

Comment:
You may not create a cross-dependency between yoiur cmdlet and the policy insights management library - the same solutions outlined above would work (haveing a captured, common version of policy insights, or moving these cmdlets into the PolicyInsights module.